include/mbedtls/config.h

```diff
@@ -1744,6 +1744,25 @@
  */
 //#define MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION
 
+/**
+ * \def MBEDTLS_X509_TRUSTED_CERTIFICATE_CALLBACK
+ *
+ * If set, this enables the X.509 API `mbedtls_x509_crt_verify_with_cb()`
```

Comment:
Note: if/when renaming this function, this occurrence needs to be renamed as well.

Reply:
Changed